scripts/mixit_batch.sh

```diff
@@ -5,6 +5,8 @@ n_threads=${N_THREADS:-8}
 
 species=${1:?Please provide a species name.}
 
+gsutil -m cp -r gs://birdclef-2023/data/raw/birdclef-2022/train_audio/${species} ~/birdclef-2023/data/raw/birdclef-2022/train_audio/
```

Comment:
Maybe make this relative to the repository root instead of relative to the home directory. `gsutil -m rsync` will also do the smart thing and not download the dataset if it already exists, otherwise I think `gcloud storage cp` is actually faster based on https://cloud.google.com/blog/products/storage-data-transfer/new-gcloud-storage-cli-for-your-data-transfers